sympy/functions/elementary/trigonometric.py

```diff
@@ -717,6 +717,7 @@ def _fermatCoords(n):
             # if n can be factored in terms of Fermat primes with
             # multiplicity of each being 1, return those primes, else
             # False
+            from sympy import chebyshevt
```

Comment:
Looks like https://github.com/sympy/sympy/pull/9293/files#diff-87ffbd4ee11461f9eb8261813a355b8cR730 is untested. Perhaps, we should also add a test.

This should have gone in another PR.